system-int-test/test/cmr/system_int_test/search/facets/facet_responses.clj

```diff
@@ -122,7 +122,46 @@
        :links
        {:apply
         "http://localhost:3003/collections.json?page_size=0&include_facets=v2&processing_level_id_h%5B%5D=PL1"},
-       :has_children false}]}]})
+       :has_children false}]}
+    {:title "Measurements",
+     :type "group",
+     :applied false,
+     :has_children true,
+     :children
+     [{:title "Measurement1",
+       :type "filter",
+       :applied false,
+       :count 2,
+       :links
+       {:apply
+        "http://localhost:3003/collections.json?page_size=0&include_facets=v2&variables_h%5B0%5D%5Bmeasurement%5D=Measurement1"},
+       :has_children true,
+       :children
+       [{:title "Variable1",
```

Comment:
Sorry if you get this comment twice - I don't think it made it the first time.

If "Measurement1" is not applied we should not display its children (so "Variable1" should not be returned).

So we should have a test that if measurement is not applied, variables are not returned, and if measurement is applied the variables below that measurement are applied, but not variables below other not applied measurements.